client/static/html/results.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Results</title>
+    <link rel="stylesheet" href="../css/results.css">
+    <script defer src="/client/js/results.js"></script>
+</head>
+<body>
+    <header>
+        <div id="header-nav-bar">
+            <div id="left-header-nav-bar">
+                <img id="google" src="https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png" alt="Google logo">
+                <input type="text" placeholder="">
```

Comment:
Could've added buttons but would only be for aesthetic reasons, everything else is good